frontend/web/components/pages/AnimeDetails.js

```diff
@@ -28,25 +29,56 @@ class AnimeDetails extends React.Component {
   render () {
     const { animeCache } = this.props;
     let content = null;
-    let details = animeCache[this.props.params.id];
+    let details = animeCache[this.props.params.id]
     if (details) {
-      console.log('got details');
-      let averageScore = (Math.round(parseInt(details.average_score, 10)))/10;
-      let description = details.description.replace(/<[^>]*>/ig, "");
+      console.log('got details')
+      console.log(details);
+      let averageScore = (Math.round(parseInt(details.average_score, 10)))/10
+      let description = details.description.replace(/<[^>]*>/ig, "")
+      let characters = []
+      if (details.characters.length > 0) {
+        characters = details.characters.slice(0, 7)
+          .map((charDetail, i) => {
+            return <li className="character-card" key={i}>
```

Comment:
Let's abstract the view code for this into a separate component called character. You can just put it at the top of this file for now, but if we ever need to use that component again all we need to do is give it the character object and your component will do everything for us. We can use the stateless component syntax for this so:
```js
const Character = ({ name_first, otherProp }) => (
  <div>...</div>
)
```